mujoco_warp/_src/types.py

```diff
@@ -219,10 +219,16 @@ class GeomType(enum.IntEnum):
   CYLINDER = mujoco.mjtGeom.mjGEOM_CYLINDER
   BOX = mujoco.mjtGeom.mjGEOM_BOX
   MESH = mujoco.mjtGeom.mjGEOM_MESH
+  SDF = mujoco.mjtGeom.mjGEOM_SDF
   # unsupported: HFIELD,
   # NGEOMTYPES, ARROW*, LINE, SKIN, LABEL, NONE
 
 
+class SDFType(enum.IntEnum):
```

Comment:
I think SDFType can be removed from the engine to the test as it's something only the specific user function supplied by the user knows about.